Adding Elixir with its Jason Parser, which seems to run ∞ :)

RobertDober opened the pull request:
Instructions:

        cd parser_elixir
        mix deps.get
        mix escript.build
        cd ..
        ./test_parser.sh ./parser_elixir/parser_elixir

RobertDober committed:
Adding Elixir with its Jason Parser, which seems to run ∞ :)

Instructions:

        cd parser_elixir
        mix deps.get
        mix escript.build
        cd ..
        ./test_parser.sh ./parser_elixir/parser_elixir

lovasoa commented:
Great, thank you ! 
Can you add it to our `.travis.yml` ?

lovasoa commented on parser_elixir/test/parser_elixir_test.exs:
```diff
@@ -0,0 +1,8 @@
+  doctest ParserElixir
+
+  test "greets the world" do
+    assert ParserElixir.hello() == :world
```
is that needed?

RobertDober replied:
good catch, will go away

lovasoa commented on .travis.yml:
```diff
@@ -76,3 +76,8 @@ matrix:
+
+    - language: elixir
+      script:
+        - mix escript.build
```
don't you need to cd into the project's directory?

RobertDober replied:
WIP I need to do this even before script so that mix can be executed, and I am discovering pitfalls all the way long, but thx for your help :blush:

I will also try to get a modern Elixir version, 1.0.2 is probably not cutting it with the `jason`.

So this will take some time :sob:

lovasoa commented on parser_elixir/README.md:
```diff
@@ -0,0 +1,21 @@
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `parser_elixir` to your list of dependencies in `mix.exs`:
```
is that relevant?

RobertDober replied:
this is the standard mix scaffolded README I guess it is only noise in this context, I'll remove it too.

RobertDober committed:
parser_elixir

   - .travis.yml for elixir
   - tests removed

RobertDober committed:
parser_elixir

   - .travis.yml for elixir
   - README removed

RobertDober committed:
parser_elixir

   - .travis.yml specifying necessary elixir version, install hex locally with --force, explicit mix deps.get

RobertDober commented:
I hope I have addressed all issues...

lovasoa commented:
Thank you!
